roles/openshift_loadbalancer_facts/meta/main.yml

```diff
@@ -10,4 +10,5 @@ galaxy_info:
     versions:
     - 7
 dependencies:
-- role: openshift_facts
+  - role: openshift_facts
+  - role: nuage_common   
```

Comment:
Not sure how I feel about adding nuage_common to the dependencies for openshift_load_balancer_facts, at a minimum I think it should be conditionalized. @abutcher thoughts?